tests/test_working_copy_postgis.py

```diff
@@ -801,3 +801,67 @@ def test_checkout_custom_crs(
                 "    },",
                 "  ]",
             ]
+
+
+def test_import_from_non_postgis_db(cli_runner, tmp_path, chdir):
```

Comment:
should rename this file (`postgis` -> `postgres`) or put this test in its own file